### PR TITLE
New package: OpenSC.OpenSC version 0.23.0

### LINKED ISSUE #123939: [Package Request]: OpenSC
### How can we help?

I would like someone else to build the manifest.

### Please read and ensure the following

- [X] The installer meets the above requirements

### Please provide the following information

Download Page Url: https://github.com/OpenSC/OpenSC/releases
Publisher: https://github.com/OpenSC/OpenSC/wiki OpenSC Team
Package Name: OpenSC
Description: Open Source smartcard driver for Windows
Package Version: 0.23.0
Installer URL: https://github.com/OpenSC/OpenSC/releases/download/0.23.0/OpenSC-0.23.0_win64.msi

### PR DESCRIPTION
**Pull request by BrandonWanHuanSheng**
Resolves #123939 

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/123971)

### CODE REVIEW

**Comment by BrandonWanHuanSheng** [comment]
I add /norestart because the short description actually saying that application having a driver.

File: manifests/o/OpenSC/OpenSC/0.23.0/OpenSC.OpenSC.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  Custom: /norestart
+UpgradeBehavior: install
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/OpenSC/OpenSC/releases/download/0.23.0/OpenSC-0.23.0_win64.msi
+  InstallerSha256: A663FDDD05F5B286E9CD48AA31D0ACFF1C2C93D3CBC76A08BA03056036F74B2F
+  InstallModes:
+  - interactive
+  - silent
+  - silentWithProgress
```
Comment (mdanish-kh) [inline manifests/o/OpenSC/OpenSC/0.23.0/OpenSC.OpenSC.installer.yaml:19]: This should be moved up at the root level

File: manifests/o/OpenSC/OpenSC/0.23.0/OpenSC.OpenSC.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,26 @@
+# Created using wingetcreate 1.5.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: OpenSC.OpenSC
+PackageVersion: 0.23.0
```
Comment (mdanish-kh) [inline manifests/o/OpenSC/OpenSC/0.23.0/OpenSC.OpenSC.installer.yaml:5]:
```suggestion
PackageVersion: 0.23.0.0
```

File: manifests/o/OpenSC/OpenSC/0.23.0/OpenSC.OpenSC.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,24 @@
+# Created using wingetcreate 1.5.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: OpenSC.OpenSC
+PackageVersion: 0.23.0
```
Comment (mdanish-kh) [inline manifests/o/OpenSC/OpenSC/0.23.0/OpenSC.OpenSC.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 0.23.0.0
```

File: manifests/o/OpenSC/OpenSC/0.23.0/OpenSC.OpenSC.yaml [hunk 4]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.5.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.5.0.schema.json
+
+PackageIdentifier: OpenSC.OpenSC
+PackageVersion: 0.23.0
```
Comment (mdanish-kh) [inline manifests/o/OpenSC/OpenSC/0.23.0/OpenSC.OpenSC.yaml:5]:
```suggestion
PackageVersion: 0.23.0.0
```

File: manifests/o/OpenSC/OpenSC/0.23.0/OpenSC.OpenSC.locale.en-US.yaml [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+Publisher: OpenSC
+PublisherUrl: https://github.com/OpenSC
+PublisherSupportUrl: https://github.com/OpenSC/OpenSC/issues
+PackageName: OpenSC
+PackageUrl: https://github.com/OpenSC/OpenSC/releases
+License: LGPL-2.1 License
+LicenseUrl: https://github.com/OpenSC/OpenSC/blob/master/COPYING
+ShortDescription: Open Source smartcard driver for Windows
+Description: OpenSC provides a set of libraries and utilities to work with smart cards. Its main focus is on cards that support cryptographic operations, and facilitate their use in security applications such as authentication, mail encryption and digital signatures. OpenSC implements the standard APIs to smart cards, e.g. PKCS#11 API, Windows’ Smart Card Minidriver and macOS CryptoTokenKit.
+Moniker: OpenSC
```
Comment (mdanish-kh) [inline manifests/o/OpenSC/OpenSC/0.23.0/OpenSC.OpenSC.locale.en-US.yaml:16]:
```suggestion
Moniker: opensc
```

The general convention is that moniker should be written in lowercase

File: manifests/o/OpenSC/OpenSC/0.23.0/OpenSC.OpenSC.locale.en-US.yaml [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+Tags:
+- OpenSC
```
Comment (mdanish-kh) [inline manifests/o/OpenSC/OpenSC/0.23.0/OpenSC.OpenSC.locale.en-US.yaml:18]:
```suggestion
- c
- security
- smartcard
- pkcs11
- tokend
- minidriver
- opensc
```

The general convention is that tags should be written in lowercase. Suggested some more tags from https://github.com/OpenSC/OpenSC

File: manifests/o/OpenSC/OpenSC/0.23.0/OpenSC.OpenSC.locale.en-US.yaml [hunk 7]
```diff
@@ -0,0 +1,24 @@
+# Created using wingetcreate 1.5.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: OpenSC.OpenSC
+PackageVersion: 0.23.0
+PackageLocale: en-US
+Publisher: OpenSC
```
Comment (mdanish-kh) [inline manifests/o/OpenSC/OpenSC/0.23.0/OpenSC.OpenSC.locale.en-US.yaml:7]:
```suggestion
Publisher: OpenSC Project
```
Must match Apps&Features / Control Panel

File: manifests/o/OpenSC/OpenSC/0.23.0/OpenSC.OpenSC.locale.en-US.yaml [hunk 8]
```diff
@@ -0,0 +1,24 @@
+# Created using wingetcreate 1.5.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: OpenSC.OpenSC
+PackageVersion: 0.23.0
+PackageLocale: en-US
+Publisher: OpenSC
+PublisherUrl: https://github.com/OpenSC
+PublisherSupportUrl: https://github.com/OpenSC/OpenSC/issues
+PackageName: OpenSC
```
Comment (mdanish-kh) [inline manifests/o/OpenSC/OpenSC/0.23.0/OpenSC.OpenSC.locale.en-US.yaml:10]:
```suggestion
PackageName: OpenSC smartcard framework
```
To match Apps&Features / Control Panel

**Review by mdanish-kh — CHANGES_REQUESTED** [review]
After committing the suggested changes, you would also have to rename the version folder to match the new PackageVersion. The folder under `manifests/o/OpenSC/OpenSC` should be renamed from `0.23.0` -> `0.23.0.0`

A quick and easy way to do this would be here in GitHub Web by pressing the <kbd>.</kbd> (dot) key on your keyboard that'll open up the PR in VS Code Online. Navigate to Explorer menu, find the required folder and make the appropriate rename changes.

After that, stage and commit your changes from Source Control menu on the left. (The changes are automatically pushed)

**Comment by BrandonWanHuanSheng** [comment]
OK.

**Comment by BrandonWanHuanSheng** [comment]
I was unable to see the folder following.

**Comment by BrandonWanHuanSheng** [comment]
> After committing the suggested changes, you would also have to rename the version folder to match the new PackageVersion. The folder under `manifests/o/OpenSC/OpenSC` should be renamed from `0.23.0` -> `0.23.0.0`
> 
> A quick and easy way to do this would be here in GitHub Web by pressing the <kbd>.</kbd> (dot) key on your keyboard that'll open up the PR in VS Code Online. Navigate to Explorer menu, find the required folder and make the appropriate rename changes.
> 
> After that, stage and commit your changes from Source Control menu on the left. (The changes are automatically pushed)

But. I am using phone right now. So a little bit tricky to see the folder was being underline in VS Code Online. Suppose, I was touching that second folder but version was being underlines. So, why I deleted that manifest in 0.23.0 and upload 0.23.0.0

**Review by mdanish-kh — APPROVED** [review]

**Pull request merged**